### PR TITLE
Update Readme.md file

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
Updates:
-Licence added to the readme file so that the contributers who are willing to do contributions can go through it on the land site itself
- Added some sentences to increase the beauty of the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,11 +196,34 @@ to the command-line arguments of javac. To override the default generation optio
...
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
```
Comment (ttung) [inline README.md:226]: need to terminate the ``` block.

**Review by kaiwalyakoparkar — COMMENTED** [review]
Changes done as per asked

File: README.md [hunk 2]
```diff
@@ -196,11 +196,36 @@ to the command-line arguments of javac. To override the default generation optio
...
 # Privacy Policy and Terms of Use
 - [Privacy Policy](https://opensource.facebook.com/legal/privacy)
 - [Terms of Use](https://opensource.facebook.com/legal/terms)
+
+```
```
Comment (ttung) [inline README.md:231]: You closed the ``` in the wrong place.   The privacy policy and terms of use are markdown and not raw text.

**Comment by aimango** [comment]
Since we already have the full MIT license [over here](https://github.com/Instagram/ig-json-parser/blob/master/LICENSE), and it's quite a large blob to have as the main Readme, we won't be pulling in this change

**Pull request closed without merging**